docs/source/conf.py

```diff
@@ -219,6 +220,16 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'StackStormDoc'
 
+
+def previous_version(ver):
+    # XXX: on incrementing major version, minor version counter is lost!
+    major, minor = ver.split('.')
```

Comment:
I'm sorry to be importunate, but that kind of problems were solved long ago... https://github.com/rbarrois/python-semanticversion